src/kv/kv_types.h

```diff
@@ -68,6 +68,11 @@ namespace kv
     {
       return {term, version};
     }
+
+    bool operator==(const TxID& other)
```

Comment:
default would work

Reply:
As discussed, this is only available from C++20, so waiting on #2720.